packages/wrangler/src/dev.tsx

```diff
@@ -293,6 +293,8 @@ export function devOptions(yargs: CommonYargsArgv) {
 type DevArguments = StrictYargsOptionsToInterface<typeof devOptions>;
 
 export async function devHandler(args: DevArguments) {
+	await printWranglerBanner();
```

Comment:
This should still take `args.updateCheck`

Reply:
I can add, but to verify my assumption here...can `--update-check` actually ever be passed to `wrangler dev`? It is not defined in `devOptions`, so unless it inherits it from somewhere I figure it's safe to be removed from that API call?

Reply:
Oh good point! In that case completely ignore me

Reply:
`updateCheck` is set by unstable dev to avoid calling out to NPM in your tests (though this is definitely less relevant with the vitest stuff), but if this handler gets called by unstable_dev, it should probably remain to keep that behaviour

Reply:
wrangler proper doesn't use `unstable_dev`, and this is the `wrangler dev` handler, so I assume we're good?

Reply:
Sounds good to me